src/sbt-test/sbt-guardrail/scala-client-codegen-app/test

```diff
@@ -0,0 +1,2 @@
+> clean
+> package
```

Comment:
The most recent issue would have been caught by `; compile ; compile`, as already up-to-date files were not considered for compilation.